docs/feature_rgblight.md

```diff
@@ -89,6 +89,7 @@ Your RGB lighting can be configured by placing these `#define`s in your `config.
 |`RGBLIGHT_SAT_STEP`        |`17`                        |The number of steps to increment the saturation by                                                                         |
 |`RGBLIGHT_VAL_STEP`        |`17`                        |The number of steps to increment the brightness by                                                                         |
 |`RGBLIGHT_LIMIT_VAL`       |`255`                       |The maximum brightness level                                                                                               |
+|`RGBLIGHT_TIMEOUT`         |`0`                         |The time in milliseconds after which the RGB lighting will be switched off if there is no keyboard activity (like a key press or encoder tick)                                                                                                                        |
```

Comment:
I'd recommend adding a note about what it means if the value is set at zero (the default value).